yourgov/components/FormDefinitionList.tsx

```diff
@@ -85,34 +88,40 @@ export function FormDefinitionListItemDate({
 }
 
 type FormDefinitionListItemDateRangeProps = {
-	label: string;
-	from: unknown | undefined;
-	to: unknown | undefined;
+	fromLabel: string;
+	fromValue?: unknown;
+	toLabel: string;
+	toValue?: unknown;
 };
 
 export function FormDefinitionListItemDateRange({
-	label,
-	from,
-	to,
+	fromLabel,
+	fromValue,
+	toLabel,
+	toValue,
 }: FormDefinitionListItemDateRangeProps) {
 	return (
-		<SummaryListItem>
-			<SummaryListItemTerm>{label}</SummaryListItemTerm>
-			<SummaryListItemDescription>
-				{from && to ? (
-					<Fragment>
-						{format(
-							typeof from === 'string' ? parseISO(from) : (from as Date),
-							'dd/MM/yyyy'
-						)}{' '}
-						–{' '}
-						{format(
-							typeof to === 'string' ? parseISO(to) : (to as Date),
-							'dd/MM/yyyy'
-						)}
-					</Fragment>
-				) : null}
-			</SummaryListItemDescription>
-		</SummaryListItem>
+		<>
+			<SummaryListItem>
+				<SummaryListItemTerm>{fromLabel}</SummaryListItemTerm>
+				<SummaryListItemDescription>
+					{format(
+						typeof fromValue === 'string'
```

Comment:
According to the schema, this won't be a string but could be `undefined`. Are we ok to handle those two scenarios and remove the `parseISO` stuff?

Reply:
And the `as Date` type casting. Same for line 120.